workinprogress.py

```diff
@@ -236,23 +206,59 @@ def main():
     foundations_list = []
     for suit in suits:
         foundations_list.append(Foundation(suit))
```

Comment:
For simple filling of a List you should just use [list comprehension](https://python-3-patterns-idioms-test.readthedocs.io/en/latest/Comprehensions.html) for this.
```suggestion
    foundations_list = [Foundation(suit) for suit in suits]
```